test/SparqlAntlrParserTest.cpp

```diff
@@ -598,6 +598,29 @@ TEST(SparqlParser, VarOrTermGraphTerm) {
   EXPECT_THAT(varOrTerm, IsIri(nil));
 }
 
+TEST(SparqlParser, Iri) {
+  auto expectIri = [](const string& input, string iri,
+                      SparqlQleverVisitor::PrefixMap prefixMap = {}) {
+    // parse by hand here to not pollute the SparqlParserHelpers
```

Comment:
```suggestion
    // Parse "by hand" in order not to pollute the `SparqlParserHelpers`.
```

Reply:
Did the change manually in the commit. Will what happens when accepting the suggestion next time.